app/jobs/regular/push_notification.rb

```diff
@@ -28,19 +28,22 @@ def execute(args)
 
           next if push_url.blank?
 
+          uri = URI.parse(push_url)
+
+          http = FinalDestination::HTTP.new(uri.host, uri.port)
+          http.use_ssl = true
+
+          request =
+            FinalDestination::HTTP::Post.new(
+              uri.request_uri,
+              { "Content-Type" => "application/json" },
+            )
+          request.body = payload.merge(notifications: notifications).to_json
+
           begin
-            result =
-              Excon.post(
-                push_url,
-                body: payload.merge(notifications: notifications).to_json,
-                headers: {
-                  "Content-Type" => "application/json",
-                  "Accept" => "application/json",
-                },
-              )
+            response = http.request(request)
 
-            if result.status != 200
-              # we failed to push a notification ... log it
+            if response.code.to_i != 200
               Rails.logger.warn(
                 "Failed to push a notification to #{push_url} Status: #{result.status}: #{result.status_line}",
```

Comment:
`result` is an undefined variable now. @nbianca can you please follow up in a separate PR?